js/pages/concept-sets/components/tabs/conceptset-expression.js

```diff
@@ -121,7 +121,7 @@ define([
       this.conceptSetItems().forEach(conceptSetItem => {
         conceptSetItem[key](!areAllSelected);
       })
-      await this.conceptSetStore.resolveConceptSetExpression();
```

Comment:
this one became redundant after fix

Reply:
The reason why we don't want to put the subscription to the observer inside the concept set store is that we don't really need to constantly resolve the concept set expression unless someone is going to be 'looking' at it.  That's why we attach the subscription in the UI components, but also have to do a 'resolve' before we hookup the subscription because we don't know if it is up to date.

You removed the subscription in the 'pages/concept-set' but not from the embedded UI (like cohort definitions, cohort characterization, ethich both use the 'shared' concept set editor UI...the repository concept set works slightly differently).   My suggestion is:  take the subscription out of concept set store, and put the resolveConceptSet expression and subscription to the conceptSetStore.observer back.